src/cli/argument-parser.js

```diff
@@ -27,7 +26,7 @@ const DESCRIPTION = dedent(`
     To run tests against all installed browsers, use the "all" alias.
 
     To use a remote browser connection (e.g. to connect a mobile device), specify "remote" as a browser alias. If you
-    need to connect multiple devices, prefix the alias with the number of browsers you want to connect (e.g., "3remote").
+    need to connect multiple devices, append the colon and the number of browsers you want to connect (e.g., "remote:3").
```

Comment:
```
To use a remote browser connection (e.g., to connect a mobile device), specify "remote" as the browser alias. If you need to connect multiple devices, add a colon and the number of browsers you want to connect (e.g., "remote:3").
```
just from the correctors.

>  add a colon and the number

that's from me

> (e.g.,

a comma after `e.g.` - from the correctors